### PR TITLE
Update nvidia-driver to support Tesla Volta cards

### PR DESCRIPTION
**Pull request by rbrthogan**
Version bump of nvidia-driver to 384.81 support Tesla V100 cards used in AWS p3 instances

### CODE REVIEW

File: build/setup.d/95-nvidia.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # See https://github.com/NVIDIA/nvidia-docker/wiki/Deploy-on-Amazon-EC2
 # But we're using the run script to not depend on build tools
 DRIVER_ARCH="Linux-x86_64"
-DRIVER_VERSION="375.26"
+DRIVER_VERSION="384.81"
 DRIVER_FILENAME="NVIDIA-${DRIVER_ARCH}-${DRIVER_VERSION}.run"
 DRIVER_CHECKSUM="d60819b2e377398c7296999ab5e7c1a4"
```
Comment (aermakov-zalando) [inline build/setup.d/95-nvidia.sh]: The hash should be updated as well.

File: build/setup.d/95-nvidia.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # See https://github.com/NVIDIA/nvidia-docker/wiki/Deploy-on-Amazon-EC2
 # But we're using the run script to not depend on build tools
 DRIVER_ARCH="Linux-x86_64"
-DRIVER_VERSION="375.26"
+DRIVER_VERSION="384.81"
```
Comment (aermakov-zalando) [inline build/setup.d/95-nvidia.sh]: http://us.download.nvidia.com/XFree86/Linux-x86_64/384.81/NVIDIA-Linux-x86_64-384.81.run leads to a 404, are you sure the version number is correct?

Comment (rbrthogan) [reply build/setup.d/95-nvidia.sh]: hmmm, ya you're right. I just found the version number here http://www.nvidia.com/download/driverResults.aspx/124728/en-us the download link appears to be different

Comment (rbrthogan) [reply build/setup.d/95-nvidia.sh]: it appears from http://www.nvidia.com/object/unix.html that 384.90 is the latest version that supports to same cards as 375.26.

Oddly, the product page seems to say that neither supports Tesla cards, despite having used 375.26 with tesla K80s.

*TLDR* I think it should be 384.90, I will update

File: build/setup.d/95-nvidia.sh (hunk 1 above)
Comment (rbrthogan) [reply build/setup.d/95-nvidia.sh]: will update

**Comment by chinoxyz** [comment]
I am wondering what is the status of this. Is this going to be merged?

**Comment by Raffo** [comment]
Can you fix the build as well?

**Comment by aermakov-zalando** [comment]
I've fixed the build [here](https://github.com/zalando-stups/taupage/pull/479), sorry for the delay.

**Comment by aermakov-zalando** [comment]
Reopening to trigger Travis.

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:
